### PR TITLE
Line break added between ticketing input and buttons

### PR DESCRIPTION
**Pull request by Saumyachitransh01**
Screenshot from 2020-11-06 09-24-47
Adding a line break for a more clear look.
Below ticketing system input field.

Screenshot from 2020-11-06 09-21-21

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia).
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The acceptance, integration, unit tests and linter pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

### CODE REVIEW

**Review by iamareebjamal — CHANGES_REQUESTED** [review]
unnecessary changes

File: app/templates/components/forms/wizard/basic-details-step.hbs [hunk 1]
```diff
@@ -1,246 +1,182 @@
-<form class="ui form {{if this.isLoading 'loading'}}" autocomplete="off" {{action 'move' 'forward' on='submit' preventDefault=true}} novalidate>
+<form class="ui form {{if this.isLoading 'loading'}}" autocomplete="off"
+  {{action 'move' 'forward' on='submit' preventDefault=true}} novalidate>
```
Comment (maze-runnar) [inline app/templates/components/forms/wizard/basic-details-step.hbs:2]: not needed

File: app/templates/components/forms/wizard/basic-details-step.hbs [hunk 2]
```diff
@@ -1,246 +1,182 @@
...
-      @value={{this.data.event.name}} />
+    <Input @type="text" @id="name" @value={{this.data.event.name}} />
   </div>
   <div class="ui secondary pointing menu">
     {{#each this.locationMenuItems as |item|}}
-      <a role="button" href="#" class="{{if (eq item this.selectedLocationType) 'active'}} item" onclick={{action (mut this.selectedLocationType) item}}>
-        {{item}}
-      </a>
+    <a role="button" href="#" class="{{if (eq item this.selectedLocationType) 'active'}} item"
+      onclick={{action (mut this.selectedLocationType) item}}>
```
Comment (maze-runnar) [inline app/templates/components/forms/wizard/basic-details-step.hbs:10]: on line 5 you have made a multiline tag to a single line and here a single line to multiline?

Comment (maze-runnar) [reply app/templates/components/forms/wizard/basic-details-step.hbs:10]: make only the change that is required.

**Pull request closed without merging**